### PR TITLE
easyTableNavigator 2.2

### PR DESCRIPTION
**Pull request by Oreonan**
### Release information
- Name (Example, Add-on Updater): 
easyTableNavigator
- Author (example, Your Name): 
Corentin Bacqué-Cazenave, Joseph Lee
- Repo (example, https://github.com/yourname/addonName): 
https://github.com/Oreonan/easyTableNavigator-1
- Version (example, 21.07): 
2.2
- Update channel (example, stable, dev or stable and dev): 
stable and dev
- NVDA compatibility (example, 2021.1 and beyond): 
2019.3 to 2021.1

## Changes for 2.2

* Update documentation style from addons template
* First translated version

### CODE REVIEW

**Review by nvdaes — COMMENTED** [review]
@Oreonan, we have to review the url and manifest. So, I suggest you to attach the binary on a GitHub release for ease of review. Also, I would mention @josephsl as an author, though he is not currently a maintainer, but he's the first creator of this add-on, and the template for PRs against this repo, at this moment, suggest to mention authors, not maintainers. Just for reference.
Thanks

**Comment by Oreonan** [comment]
I've added @josephsl to authors, sorry, I forgot to add him yesterday...
The release with binary can be find here:
https://github.com/Oreonan/easyTableNavigator-1/releases/tag/v2.2

File: get.php [hunk 1]
```diff
@@ -48,8 +48,8 @@
 	"emo" => "https://github.com/nvdaes/emoticons/releases/download/14.2/emoticons-14.2.nvda-addon",
 	"emo-dev" => "https://github.com/nvdaes/emoticons/releases/download/14.2/emoticons-14.2.nvda-addon",
 	"emo-o" => "https://github.com/nvdaes/emoticons/releases/download/6.5/emoticons-6.5.nvda-addon",
-	"etn" => "easyTableNavigator-2.1.1.nvda-addon",
-	"etn-dev" => "easyTableNavigator-2.1.1.nvda-addon",
+	"etn" => "easyTableNavigator-2.2.nvda-addon",
```
Comment (nvdaes) [inline get.php:51]:
```suggestion
	"etn" => "https://github.com/Oreonan/easyTableNavigator-1/releases/download/v2.2/easyTableNavigator-2.2.nvda-addon",
```

File: get.php [hunk 2]
```diff
@@ -48,8 +48,8 @@
...
+	"etn-dev" => "easyTableNavigator-2.2.nvda-addon",
```
Comment (nvdaes) [inline get.php:52]:
```suggestion
	"etn-dev" => "https://github.com/Oreonan/easyTableNavigator-1/releases/download/v2.2/easyTableNavigator-2.2.nvda-addon",
```

**Review by nvdaes — CHANGES_REQUESTED** [review]
Thanks. Can you use the GitHub URL instead of binaries?

**Review by nvdaes — APPROVED** [review]
Thanks. I advice, instead of claiming for NVDA compatibility, in the add-on documentation, from 2019.3 to 2021.1, reflect: 2019.3 and beyond, since this will be compatible with 2021.2 and later versions

**Comment by nvdaes** [comment]
I closed this accidentally, then reopened awaiting a review from @feerrenrut

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**